src/telegram/process_update_task.rs

```diff
@@ -39,24 +43,66 @@ impl ProcessUpdateTask {
         log::info!("Received a message {:?}", self.update);
 
         let api = AsyncApi::new(&RUST_TELEGRAM_BOT_TOKEN);
+        let db_controller = DbController::new().await?;
```

Comment:
I think creating a new db controller for every command is not the best approach.

Since `fang` already has db pool, it can be re-used here

Reply:
we will re-think later